src/main/java/org/intellij/sdk/codesync/database/Database.java

```diff
@@ -93,6 +107,11 @@ public static ArrayList<HashMap<String, String>> runQuery(String query) throws S
 
     public static void executeUpdate(String query) throws SQLiteDBConnectionError, SQLiteDataError{
 
+        if(!isConnected()){
```

Comment:
@syedgulahmed Have we added test for this? i.e. close connection explicilty and then run `executeUpdate` and verify it reconnects correctly?

Reply:
Though, I have tested it manually but we don't have test coverage for this line. There is different CONNECTION_STRING for unit test and I would have to create extra parameter in execuateUpdate/runQuery to do it.

Reply:
ok, why do we have different CONNECTION_STRING for tests and actual code?

Reply:
`CONNECTION_STRING` contains file path of db file and we different files for dev, prod and testing.

Reply:
We use different base directory, `~/.codesync`, `~/.codesync-dev` and connection string is relative to the base path. right?
Why do we need to create extra parameters for tests?